.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 *         @Xavrax @mohitpubnub @kleewho @marcin-cebo
 .github/* @parfeon @Xavrax @mohitpubnub @kleewho @marcin-cebo
-**/README.md @techwritermat @kazydek @michaljolender
```

Comment:
@Xavrax are you sure about that? What if there will be `CHANGELOG.md` which pretty often is present in repos?

Reply:
Ahhh. you're right. I will come back to all readmes.

Reply:
I reverted and now only Readme.md files are taken into account.